### PR TITLE
docs: add base dataset samples for the automl ga

### PR DESCRIPTION
**Pull request by nnegrey**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-automl/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: samples/delete_dataset.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/**
```
Comment (JustinBeckwith) [inline samples/delete_dataset.js]: We recently updated all the license headers to be consistent.  Could I trouble you to copy from an existing template?

File: samples/test/export_dataset.test.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    const bucketName = `${projectId}-automl-translate`;
+
+    const storageClient = new Storage();
+    const options = {
+      prefix: prefix,
+    };
+    const [files] = await storageClient
+      .bucket(`gs://${bucketName}`)
+      .getFiles(options);
+    files.forEach(file => {
```
Comment (JustinBeckwith) [inline samples/test/export_dataset.test.js]: This is going to exit before the operation is complete.  You want something like:

```js
await Promise.all(
  files.map(file => 
      storageClient
        .bucket(`gs://${bucketName}`)
        .file(file.name)
        .delete();
);
```

Comment (nnegrey) [reply samples/test/export_dataset.test.js]: Can you delete a whole folder at once in Node.js with Cloud Storage?
I can't in other languages, but that would be great.

Comment (bcoe) [reply samples/test/export_dataset.test.js]: @jkwlui would be a good person to ask, I know we can upload a folder, but I don't believe we can delete yet.

**Pull request merged**